RecoMET/METFilters/python/EcalBadCalibFilter_cfi.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+ 
+EcalBadCalibFilter = cms.EDFilter(
+  "EcalBadCalibFilter",
+ 
+  # use this if using AOD:
+ 
+  # the EB rechit collection found in AOD
+  EBRecHitSource = cms.InputTag('reducedEcalRecHitsEB'), 
+  # the EE rechit collection found in AOD
+  EERecHitSource = cms.InputTag('reducedEcalRecHitsEE'),
+ 
+ 
+  # use this if using MINIAOD:
+ 
+  # the EB rechit collection found in MINIAOD
+  # EBRecHitSource = cms.InputTag('reducedEgamma','reducedEBRecHits'), 
+  # the EE rechit collection found in MINIAOD
+  # EERecHitSource = cms.InputTag('reducedEgamma','reducedEERecHits'),
+ 
+ 
+ 
+  # minimum rechit et to flag as bad:  EB 
+  EBminet        = cms.double(50.),
+  # minimum rechit et to flag as bad:  EE 
+  EEminet        = cms.double(50.),
+  # DetId of bad channel: EE
+  baddetEB        = cms.vuint32(0),
+  # DetId of bad channel: EE
+  baddetEE        = cms.vuint32(872439604,872422825,872420274,872423218,
+                                872423215,872416066,872435036,872439336,
+                                872420273,872436907,872420147,872439731,
+                                872436657,872420397,872439732,872439339,
+                                872439603),                                  
```

Comment:
what is the scope of this set of constants?
Does this apply to Run1? to 2015, 2016 or 2017?
If the answer varies, please use appropriate sets configured via era modifiers

Reply:
Hi @slava77 these numbers apply for 2016 and 2017. Could please give use some hints on how to do it ? Thank you.

Reply:
A related question: does this apply to 
- the 2017 data rereco with 92X
- the 2017 data reco with 94X [this release]

I somehow thought that the intercalibrations were fixed recently

Reply:
@slava77 The rate of problematic events in 07Aug17 dropped significantly however there are still some remaining events with intercalibration issues. The list of det-ids in this pr includes both the old detids and the new discovered one.

Reply:
OK, so it sounds like the upcoming Oct17 reco may need no filter or may need a different filter.
Also, consistent application to the past processing may be difficult because the filter ultimately makes sense only on AOD (and even miniAOD) inputs that were produced from RECO step with bad GT used in 2016 legacy and some in 07Aug17 rerecos. So, era modifiers mentioned earlier are not really applicable.

May I suggest to rename this from the generic "EcalBadCalibFilter" to something with a specific list in mind
e.g. ```ecalBadCalibSep2017ListFilter``` or something more creative 
(please also mind the capitalization style for the first letter)